test/gui/tst_sharing/test.feature

```diff
@@ -133,3 +133,16 @@ Feature: Sharing
             | path        | /simple-folder |
             | name        | Public link    |
         And the public should not be able to download the file "lorem.txt" from the last created public link by "Alice" on the server
+
+    Scenario: Added test for shareing to a group to which the file is already shared
+        Given the setting "shareapi_auto_accept_share" on the server of app "core" has been set to "yes"
+		And group "grp1" has been created on the server
```

Comment:
fix indentation
```suggestion
	And group "grp1" has been created on the server
```

Reply:
Looks good in Squish. the indentation looks bad in the GitHub only.